src/pages/gen1/[platform]/tools/cli/reference/files/index.mdx

```diff
@@ -117,25 +117,15 @@ Because one category might create multiple services within one project (e.g. the
 
 The metadata for each service is first logged into the meta file after the `amplify <category> add` command is executed, containing some general information that indicates one service of the category has been added locally. Then, on the successful execution of the `amplify push` command, the `output` object will be added/updated in the service's metadata with information that describes the actual cloud resources that have been created or updated.
 
-### aws-exports.js
-
-> Manual edits okay: NO
-
-> Add to version control: NO
-
-This file is generated only for JavaScript projects. It contains the consolidated outputs from all the categories and is placed under the `src` directory specified during the `init` process. It is updated after `amplify push`.
-
-This file is consumed by the [Amplify](https://github.com/aws-amplify/amplify-js) JavaScript library for configuration. It contains information which is non-sensitive and only required for external, unauthenticated actions from clients (such as user registration or sign-in flows in the case of Auth) or for constructing appropriate endpoint URLs after authorization has taken place.
-
 ### amplifyconfiguration.json
 
 > Manual edits okay: NO
 
 > Add to version control: NO
 
-This file is generated by Amplify CLI for Swift and Android projects. It contains the consolidated outputs from all the categories and is updated after `amplify push`. It is consumed by the [Amplify Swift](https://github.com/aws-amplify/amplify-swift/) and [Amplify Android](https://github.com/aws-amplify/amplify-android) libraries for configuration.
+This file is generated by Amplify CLI for JavaScript or Typescript, Swift and Android projects. It contains the consolidated outputs from all the categories and is updated after `amplify push`. It is consumed by the [Amplify JS](https://github.com/aws-amplify/amplify-js), [Amplify Swift](https://github.com/aws-amplify/amplify-swift/) and [Amplify Android](https://github.com/aws-amplify/amplify-android) libraries for configuration.
```

Comment:
```suggestion
This file is generated by Amplify CLI for JavaScript, Swift and Android projects. It contains the consolidated outputs from all the categories and is updated after `amplify push`. It is consumed by the [Amplify JS](https://github.com/aws-amplify/amplify-js), [Amplify Swift](https://github.com/aws-amplify/amplify-swift/) and [Amplify Android](https://github.com/aws-amplify/amplify-android) libraries for configuration.
```

rm'd "TypeScript" callout to avoid confusion with Gen 1's project types (JavaScript, Android, Swift, Flutter)